### ISSUE TITLE
livequery functionality

### ISSUE THREAD

**Issue by julien9999** [E2010/android-parse-module-titanium-3-5#2]
Hi, does it support live query ?
Thanks

**Comment by E2010** [comment]
No. Sorry. This is a module I used on a project of my company. Only the functions we need in our project (pull request related) completed.
